com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/pywrapfunction.cgt

```diff
@@ -14,24 +14,30 @@
     PyObject * appCallable = 
       streamsx::topology::Splpy::loadFunction(<%=$pyModule%>, <%=$pyCallableName%>);
 
+	<% if ($pystyle eq 'string' or $pystyle eq 'message') { %>
```

Comment:
I don't think this is correct. If the passed in callable is a class and pystyle is string then _function will be set to the module, right?

I.e. the code does not work when pystyle is string and pyCallable is set.